website/content/terminal/crypto/onchain/btcblockdata/_index.md

```diff
@@ -0,0 +1,15 @@
+```
+usage: btcblockdata     [--blockhash BLOCKHASH]
+                        [--export {csv,json,xlsx}] [-h]
+```
+
+Displays json data for a single block. [Source: https://api.blockchain.info/]
+
+```
+arguments:
+  --blockhash BLOCKHASH         Hash of the block you are targeting  
+  --export {csv,json,xlsx}      Export dataframe data to csv,json,xlsx file (default: )
+  -h, --help                    Show this help message (default: False)
+```
+
+![](../../../../../../../../../Dropbox/Screenshot 2022-10-26 000048.jpg)
```

Comment:
the image is only locally on your computer, can you use the image that's in the pr? - https://user-images.githubusercontent.com/57280701/201509706-a7c94df8-e15e-4b99-97bd-4cefa52edb02.jpg

Reply:
Done